### PR TITLE
Add a new canned response

### PR DESCRIPTION
**Pull request by magogre**
Added a new decline reason. This will be useful to decline the wiki requests whose database name is already taken by wikis which aren't deleted or active but closed.

### CODE REVIEW

**Comment by DarkMatterMan4500** [comment]
Hmmm?

**Review by ugochimobi — APPROVED** [review]
Nice concept, code looks fine.

File: LocalSettings.php [hunk 1]
```diff
@@ -823,6 +823,7 @@
...
 			'Database exists (wiki still active)' => 'Wiki database name and subdomain already exist. Please visit the local wiki and contribute there. Please reach out to any local bureaucrat to request any permissions if you require them. If bureaucrats are not active on the wiki after a reasonable period of time, please start a local election and ask a Steward to evaluate it on the Stewards\' noticeboard. Thanks.',
 			'Database exists (wiki already deleted)' => 'Wiki database name already exists, but the wiki itself has already been deleted in accordance with the Dormancy Policy. I will request a Steward to undelete it for you. When it has been undeleted and reopened, please visit the local wiki and ensure you make at least one edit or log action every 45 days. Wikis are only deleted after 6 months of complete inactivity; if you require a Dormancy Policy exemption, you should review the policy and request it once your wiki has at least 40-60 content pages. Thank you.',
+                        'Database exists (wiki closed)' => 'Wiki database name already exists, but the wiki is closed due to inactivity. Please visit the Requests for adoption page to request to reopen the wiki.',
```
Comment (Universal-Omega) [inline LocalSettings.php:826]: Wikis can be closed for reasons besides inactivity I believe, making this somewhat inaccurate.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (magogre) [reply LocalSettings.php]: I agree. I have tweaked the response, now.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**